lib/generators/thredded/install/install_generator.rb

```diff
@@ -50,43 +50,6 @@ def copy_views_and_assets
           'app/assets/images/thredded',
           'vendor/assets/images/thredded'
       end
-
-      def add_dependencies
-        return if no_gems_to_install? || !options.theme?
-
-        line_in_gemfile = in_gemfile?('ruby') ? /^ruby .*$/ : /^source.*$/
-        inject_into_file 'Gemfile', gems, after: line_in_gemfile
-        notify_of_installation
-      end
-
-      private
-
-      def no_gems_to_install?
-        gems.chomp.empty?
-      end
-
-      def gems
-        gems = "\n"
-        gems << "\ngem 'bourbon'"       unless in_gemfile?('bourbon')
-        gems << "\ngem 'neat'"          unless in_gemfile?('neat')
-        gems << "\ngem 'bitters'"       unless in_gemfile?('bitters')
-        gems << "\ngem 'sprockets-es6'" unless in_gemfile?('sprockets-es6')
-
-        gems
-      end
-
-      def in_gemfile?(gem_name)
-        gemfile.include?(gem_name)
-      end
-
-      def gemfile
-        @gemfile ||= File.read('Gemfile')
-      end
-
-      def notify_of_installation
-        text = "\n\tNew gems have been added to the Gemfile. Make sure to `bundle install`.\n"
-        puts "\e[32m#{text}\e[0m"
-      end
```

Comment:
I totally get the intent of removing the work these are doing but, in the practice of implementing these on @saturnflyer's purple door project, if these asset gems aren't added to the parent app's Gemfile - bourbon, neat, bitters, et al, won't work. I had a branch where I removed them, and ... technical term here ... shit got fucked.